src/app/components/table/table.ts

```diff
@@ -5133,7 +5133,7 @@ export class ReorderableRow implements AfterViewInit {
                 [ngClass]="{ 'p-column-filter-overlay p-component p-fluid': true, 'p-column-filter-overlay-menu': display === 'menu' }"
                 [id]="overlayId"
                 [attr.aria-modal]="true"
-                role="dialog"
+                role="filterDialog"
```

Comment:
filterDialog role does not exist as an HTML Role. It breaks the accessibility of dialog.